src/library.js

```diff
@@ -2223,43 +2223,61 @@ mergeInto(LibraryManager.library, {
 
   // random.h
 
-  // TODO: consider allowing the API to get a parameter for the number of
-  // bytes.
-  $getRandomDevice: function() {
+  $initRandomDevice: function() {
```

Comment:
How about calling this `makeRandomFill`?   I'm not sure about `make` or `Device` parts of the name.

Reply:
Sure. `getRandomDevice` / `randomDevice` didn't make sense to me but didn't want to deviate too far, so renamed to `initRandomDevice`, but `makeRandomFill` does seem even better. Although I think I'd still prefer `init` prefix to make it a bit more clear it's meant for one-time initialization.

Reply:
Renamed to initRandomFill.